nav2_route/test/CMakeLists.txt

```diff
@@ -53,6 +53,17 @@ target_link_libraries(test_operations
   ${library_name} route_operations
 )
 
+# Test geojson parser
+ament_add_gtest(test_geojson_graph_file_loader
+    test_geojson_graph_file_loader.cpp
+  ../src/plugins/graph_file_loaders/geojson_graph_file_loader.cpp
```

Comment:
huh? 

Remove that. INstead in the target link libraries, just add the geojson library. See `test_collision_operation` below where it relies on a `route_operation` plugin in that library name. That way there are no fragile filepaths

Reply:
Yeah, I just did this quick a while ago to get the test up and running :).